File: jcarrollonlinev4.backend/Models/Users/UserStatsModel.cs
namespace jcarrollonlinev4.backend.Models.Users
{
    public class UserStatsModel : UserModelBase
    {
        public UserStatsModel()
        {
            UserFollowers = new UserFollowersModel();
            UsersFollowing = new UserFollowingModel();
        }
        public UserFollowersModel UserFollowers { get; set; }
        public UserFollowingModel UsersFollowing { get; set; }
    }
}

File: jcarrollonlinev4.frontend/src/models/Users/UserStatsModel.ts
import { UserFollowersModel } from "./UserFollowersModels";
import { UserFollowingModel } from "./UserFollowingModel";
import { UserModelBase } from "./UserModelBase";

export interface UserStatsModel extends UserModelBase {
  UserFollowersModel?: UserFollowersModel;
  UserFollowingModel?: UserFollowingModel;
}
